### PR TITLE
QTouchposeApplication: Fix keyWindow deprecated in iOS 13.0.

### PR DESCRIPTION
**Pull request by ShiraOzeri**

### CODE REVIEW

**Review by lior7 — APPROVED** [review]
LGTM.

File: Touchpose/Touchpose/QTouchposeApplication.m [hunk 1]
```diff
@@ -303,14 +303,17 @@ - (void)setShowTouches:(BOOL)showTouches
 {
     if (showTouches)
     {
-        if (_touchView == nil && self.keyWindow)
+        for (UIWindow *window in [UIApplication sharedApplication].windows)
```
Comment (lior7) [inline Touchpose/Touchpose/QTouchposeApplication.m]: Can you use `self` instead of `[UIApplication sharedApplication]`? (is it the same object)

**Review by lior7 — CHANGES_REQUESTED** [review]
Fix and squash.

**Comment by ShiraOzeri** [comment]
F&S

**Review by lior7 — APPROVED** [review]
LGTM.

**Comment by barakwei** [comment]
It's a good question what the solution is, you can theoretically have multiple key windows if you have multiple scenes IIUC

**Comment by lior7** [comment]
@barakwei I think so too, but in Quickart we only have one, maybe we could continue looping and assert that there is one, or just leave it as the first one we got?

**Comment by barakwei** [comment]
If it works for PF and QA feel free to merge. We'll find a proper solution later.

**Comment by lior7** [comment]
@barakwei great, but I don't have permissions.

**Pull request merged**